docs/data-prep.md

```diff
@@ -68,186 +75,331 @@ A valid metadata file must include the following fields:
 
-### Subsampling
+Find the "Location" field and start typing "North America /".
+As you type, the field will suggest more specific geographic scales.
 
-We've outlined several methods for subsampling, including builds with a focus area + genetically similar contextual sequences, in the [next section](running.md).
+![GISAID initial search interface](images/gisaid-initial-search-interface.png)
 
----
+Finish by typing "North America / USA / Washington".
+Select all strains collected between May 1 and June 1 with complete genome sequences and collection dates.
+Click the checkbox in the header row of the results display, to select all strains that match the search parameters.
 
-## Appendix: in-depth guide to the standard Nextstrain metadata fields
-**Column 1: `strain`**
+![GISAID search results for "Washington"](images/gisaid-search-results.png)
 
-This needs to match the name of a sequence in the FASTA file exactly and must not contain characters such as spaces, or `()[]{}|#><`.
-In our example we have a strain called "NewZealand/01/2020" so there should be a sequence in the FASTA file for ">NewZealand/01/2020" (sequence names in FASTA files always start with the `>` character, but this is not part of the name).
+<p style="color: #212529; background-color: #ffc107; border-color: #b6effb; padding: 1em; border-radius: .25rem;">
+GISAID limits the number of records you can download at once to 5000.
+If you need to download more records, constrain your search results to smaller windows of time by collection date and download data in these smaller batches.
+</p>
 
-**Note that "strain" here carries no biological or functional significance** and should be thought of as synonymous with sample.
+Select the "Download" button in the bottom right of the search results.
+From the resulting "Download" window, select "Input for the Augur pipeline" as the download format.
 
-**Column 2: `virus`**
+![GISAID search download window showing "Input for the Augur pipeline" option](images/gisaid-search-download-window.png)
 
-Name of the pathogen.
+Select the "Download" button and save the resulting file (named like `gisaid_auspice_input_hcov-19_2021_06_01_17.tar`) to the `data/` directory.
+This tar archive contains compressed metadata and sequences named like `1622567829294.metadata.tsv.xz` and `1622567829294.sequences.fasta.xz`, respectively.
+You can use this tar file as an input for the Nextstrain workflow, as shown below, and let the workflow extract the data for you.
+Create a new configuration file, `builds.yaml`, in the top-level of the `ncov` directory that defines your analysis or "builds".
 
-**Column 3: `gisaid_epi_isl`**
+```yaml
+# Define inputs for the workflow.
+inputs:
+  - name: washington
+    # The workflow will detect and extract the metadata and sequences
+    # from GISAID tar archives.
+    metadata: data/gisaid_auspice_input_hcov-19_2021_06_01_17.tar
+    sequences: data/gisaid_auspice_input_hcov-19_2021_06_01_17.tar
```

Comment:
I'd be designing this workflow for users to continually update by clicking through GISAID interface. I'd think easier to have users rename the downloaded file pair to something more persistent like `metadata_washington.tsv.xz` and `sequences_washington.fasta.xz` and then just be able to have `inputs/metadata` not have to be updated with new date every time someone decides to update the analysis with the latest GISAID data.

After all, files need to be renamed anyway from things like `1622567829294.metadata.tsv.xz` and so might as well not force people to modify `builds.yaml` as well.

Reply:
Thank you, @trvrb! I've fixed this in 7f376c3.